### PR TITLE
Price fix

### PR DESCRIPTION
**Pull request by dugoh**
Small fix for commerce.price()

### CODE REVIEW

**Comment by Marak** [comment]
@dugoh -

What does this fix? Could you show us a small example of the current / expected behaviors?

Thanks!

File: lib/commerce.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ var Commerce = function (faker) {
    * @param {string} symbol
    */
   self.price = function(min, max, dec, symbol) {
-      min = min || 0;
+      min = min || 1;
```
Comment (alexpts) [inline lib/commerce.js]: Why start value on default 1 better that 0?

Comment (dugoh) [inline lib/commerce.js]: I don't know, but that is what the test expects. Without the fix the test fails about 1 in 1000 times.

**Comment by dugoh** [comment]
@Marak 
If you look at line 2556 in https://travis-ci.org/Marak/faker.js/jobs/200239663 you see the output is missing a round bracket on the right.

Also the test for price has the assertion
        assert.equal((price > 0), true, "the amount should be greater than 0");
but the function defaults to a 0 minimum price causing the test to fail randomly about 1 in a 1000 times.

**Comment by epayet** [comment]
I 👍 this, I just had a case where it was generating a price of 0 and breaking things, so I had to specify a minimum to fix it: `faker.commerce.price(1)` instead of just `faker.commerce.price()`.
But I would expect the function to have more than 0 as a minimum by default.

Plus, as @dugoh mentioned, the test specifically expect the price to be greater than 0, so it can fail randomly about 1 in a 1000 times.

**Comment by Marak** [comment]
I see the issue with the tests now.

I guess it makes sense that a price should be `greater than 0` by default, although you could have a price of `0`.

**Pull request merged**